### DISCUSSION TITLE
how to process pdf documents?

### DISCUSSION THREAD

**Discussion by WuYanZhao107** [langchain-ai/langchain-extract#123] [Q&A]
I want to know langchain-extract how to process pdf documents? It's a important step for extracting structured information from pdf docements.

**Comment by eyurtsev** [comment]
This is the code that processes PDFs: https://github.com/langchain-ai/langchain-extract/blob/main/backend/extraction/parsing.py#L14

Feel free to configure for your use case. LangChain has many integrations and you can try different strategies for parsing PDFs or add parsing for other types of files (e.g., PPTX)
